app/assets/javascripts/client/inventories/invite_inventory_user.html

```diff
@@ -0,0 +1,45 @@
+<form  name="formInviteParticipant"  novalidate>
+  <alert ng-repeat="alert in alerts" type="{{alert.type}}" close="closeAlert($index)">{{alert.msg}}</alert>
```

Comment:
This function should be prefixed with the name defined in the `controllerAs` block of `inviteInventoryUser`.